types/remarkable/package.json

```diff
@@ -0,0 +1,6 @@
+{
+    "private": true,
+    "dependencies": {
+        "highlight.js": "^10.1.0"
```

Comment:
I had a similar problem, this cannot be just done by adding an updated dependency to 
DT for older version of package,
the current DT version is supporting v9, not v10:
https://github.com/jonschlinkert/remarkable/blob/v1.7.4/package.json#L74